test/specs/notification_spec.rb

```diff
@@ -2,9 +2,18 @@
 require 'pry'
 
 class Jackal::Slack::Notification
-  def post_to_slack(description, attachments)
-    $attachments = attachments
-    $description = description
+  alias_method :og_execute, :execute
```

Comment:
@chrisroberts let me know what you think of this pattern... was the easiest way i could think to patch into callback code and check if it was executed.

lol, so if it's overly complicated and there's some magic jackal way to check for the actual execution of a callback from the supervisor or something, let me know.

Otherwise, maybe we can abstract this notion at a more core level so other services can test things similarly